server.js

```diff
@@ -1115,6 +1147,12 @@ if (config.startServer) {
                 callback(null);
             });
         },
+        (callback) => {
+            util.callbackify(assets.init)(err => {
+                if (ERR(err, callback)) return;
+                callback(null);
+            });
+        },
```

Comment:
This works!
```suggestion
        async () => {
            await assets.init();
        },
```

Reply:
🤯 that's awesome! Was just pattern-matching off some other stuff in this section, should probably go back and "modernize" those too. I bet this could be even simpler:
```suggestion
        () => assets.init(),
```
since it's all just Promises under the hood.

Reply:
Update: you definitely cannot do that! The `async` library internally has a check like `fn.constructor.name === 'AsyncFunction';` - it doesn't just check if a return value is a Thenable. I'll update to use your suggestion.

Reply:
Interesting!

Of course I'd also be happy to converted the entire sequence to a/a and get rid of `async.series()` completely.